packages/loot-core/bin/build-browser

```diff
@@ -35,6 +36,9 @@ if [ $NODE_ENV == 'development' ]; then
   ln -snf "$ROOT"/../lib-dist/browser ../../desktop-client/public/kcab
 fi
 
+cp ../../../node_modules/@jlongster/sql.js/dist/sql-wasm.wasm "$PUBLIC_DIR"/sql-wasm.wasm
+cp ../../node-libofx/libofx.web.wasm "$PUBLIC_DIR"/libofx.web.wasm
```

Comment:
Just checking: did you verify that the file we had in `public` is exactly the same as the one we are copying? If it's not exactly the same - are we losing some functionality?

I wouldn't be surprised if there were some manual hacks in those files.

Reply:
Confirmed via the API that [the `libofx.*.wasm` files in `node-libofx`](https://api.github.com/repos/actualbudget/actual/git/trees/b1d7824f67d6f6f9b18c1690002dfaff9e1de1f7) and [`libofx.web.wasm` in `desktop-client`](https://api.github.com/repos/actualbudget/actual/git/trees/872e4e02c480181d33e4801d1b84043e9f8ab1ba) point to the same blob, which means they are the same file. Also confirmed the `sql-wasm.wasm` files are the same with `shasum` locally.